docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,253 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+the relay chain.
+
+Note how there are always three unincluded parablocks at all times, i.e. compared to synchronous
+backing there can be multiple unincluded parablocks (i.e. [pipelining](#pipelining)). For example,
+when P1 undergoing inclusion, P2 and P3 are undergoing backing. Collators were able to generate
+multiple unincluded parablocks because on their end they have the
+[unincluded segment](#unincluded-segments), a local storage of not included parablock ancestors that
+they can use to fetch information to build new parablocks. On the relay chain side
+[perspective parachains](#prospective-parachains) repeats the work each unincluded segment does in
+tracking candidates (as validators cannot trust the record kept on parachains).
+
+Collating on top of unincluded parents means the only time limit to generate a parablock is how long
+it takes to back it (2 seconds). The 6-second relay chain block delay includes a backing execution
+timeout (2 seconds) and some time for network latency (time it takes to gossip messages across the
+entire network). Thus, even if a collator has >2 seconds to produce a block, the validators will
+still have ~2 seconds to check it.
+
+## Terminology
+
+### Candidate Receipt
+
+Saying that a parablock has been included in a realy chain parent does not mean that the entire
+parablock is in the relay chain block. Instead, **candidate receipt** consisting of the hash of the
+parablock, state roots and ID info is placed on the parent block on the relay chain. The relay chain
+does not access the entire state of a parachain but only the values that changed during that block
+and the merkelized hashes of the unchanged values.
+
+### Pipelining
+
+Asynchronous backing is a feature that introduces
+[pipelining](https://www.techtarget.com/whatis/definition/pipelining) to the parachain block
+[generation, backing and inclusion](./learn-parachains-protocol.md). It is analogous to the logical
+pipelining of processor instruction in "traditional" architectures, where some instructions may be
+executed before others are complete. Instructions may also be executed in parallel, enabling
+multiple processor parts to work on potentially different instructions simultaneously.
+
+Bundles of state transitions represented as blocks may be processed similarly. In the context of
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, pipelining aims to increase the
+throughput of the entire network by completing the backing and inclusion steps for different blocks
+at the same time. Asynchronous backing does not just allow for pipelining within a single pipe (or
+core). It lays the foundation for a large number of pipes (~10 cores) to run for the same parachain
+at the same time. In that way we have two distinct new forms of parallel computation: having one
+block backed and one included simultaneously using one core, and having having `n` blocks backed and
+`n` blocks included simultaneously using `n` cores.
```

Comment:
nit pick, but are these actually different? how is n = 1 fundamentally different than n != 1?

Reply:
I will deleted this part